backend/src/main/java/asegroup1/api/controllers/LocationController.java

```diff
@@ -0,0 +1,27 @@
+package asegroup1.api.controllers;
+
+import asegroup1.api.models.LocationData;
+import asegroup1.api.services.location.LocationServiceImpl;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("/location")
+public class LocationController {
+
+	private LocationServiceImpl locationService;
+
+	@Autowired
+	public LocationController(LocationServiceImpl locationService) {
+		this.locationService = locationService;
+	}
+
+	@RequestMapping(value = {"","/"},method = RequestMethod.POST)
+	public @ResponseBody
+	ResponseEntity<String> post(@RequestBody LocationData location) {
```

Comment:
change `post()` to more descriptive name

Reply:
done changed to `postUpdateLocationData`